conda/meta.yaml

```diff
@@ -7,24 +7,27 @@ source:
     git_rev: v1.6.0
 
 build:
-    script: python setup.py install
+    number: 1
+    noarch: python
```

Comment:
By adding this line, you don't need separate builds/uploads to anaconda for different platforms (linux, mac and windows) or different python versions (2.7, 3.6 and 3.7).